src/test/data/XmlAddressBookStorageTest/invalidPersonAddressBook.xml

```diff
@@ -2,6 +2,7 @@
 <addressbook>
     <!-- Person with invalid name field -->
     <persons>
+    	<nric>S1234567A</nric>
```

Comment:
This line is not aligned. Same for a few more lines below.

Reply:
Tabs vs spaces, I'll fix this in a commit.